src/chart/drawers/chart-type-drawers/line.drawer.ts

```diff
@@ -33,7 +34,7 @@ export class LineDrawer implements SeriesDrawer {
 			for (let i = 1; i < visualCandles.length; i++) {
 				const prev = visualCandles[i - 1];
 				const vc = visualCandles[i];
-				const direction = vc.name;
+				const direction = nameDirection(prev.close, vc.close);
```

Comment:
We have a `defaultCandleTransformer` and `trendCandleTransformer`, looks like `trendCandleTransformer` can be used for line chart type